.gitignore

```diff
@@ -1,2 +1,4 @@
 /target
 Cargo.lock
+.DS_Store
+*/.DS_Store
```

Comment:
nit: I think the line above already applies to all directories?

Reply:
Oh sorry that might have gotten added by accident